vendor/k8s.io/klog/README.md

```diff
@@ -1,7 +1,7 @@
 klog
 ====
 
-klog is a permanent fork of https://github.com/golang/klog.
+klog is a permanent fork of https://github.com/golang/glog.
```

Comment:
Why has the vendor file changed here? We haven't re-vendored in this commit have we?

Reply:
I just used `make revendor`.